Aliases/map - move - over/5eb8d24b-d660-42e5-bbea-47a659bb0827 - Main GVAR.gvar

```diff
@@ -266,11 +259,48 @@ if c:
   #If there is a -wall
   if args.last('walls'):
    if args.last('walls') in ('none', 'None', True, 'true', 'True'):
-    walls = ""
+    walls = []
     desc.append(f"Walls reset")
    else:
-    walls = args.last('walls')
-    desc.append(f"Walls added `{walls}`")
+    inputtedWalls = args.last('walls')
+    walls.append(inputtedWalls)
+    desc.append(f"Walls added `{inputtedWalls}`")
+  if args.last('deletewalls'):
+   inputtedWalls = args.last('deletewalls')
+   walls.remove(inputtedWalls)
+   desc.append(f"Walls removed `{inputtedWalls}`")
+  #If there is a -object call
+  if args.last('object'):
+   presetObjects = load_json(get_gvar("9b15344a-1f09-43b5-b84e-f61e32e4a89a"))
+   if args.last('object') in ('none', 'None', True, 'true', 'True'):
+    objects = []
+    desc.append(f"Objects reset")
+   else:
+    foundObject = ""
+    argsObject = args.last("object").lower().split(":")
+    for object in presetObjects:
+     if argsObject[1] in object.lower():
+      foundObject = presetObjects[object]
+      break
+    if foundObject:
+     objectLocation = argsObject[0]
+     loadedObject=objectLocation+foundObject
+     objects.append(loadedObject)
+     desc.append(f"Object added `{object}:{loadedObject}`")
+    else:
+     err("Did not find object")
+  if args.last('removeobject'):
+   presetObjects = load_json(get_gvar("9b15344a-1f09-43b5-b84e-f61e32e4a89a"))     
+   foundObject = "There is no object by that name"
+   argsObject = args.last("removeobject").lower().split(":")
+   for object in presetObjects:
+    if argsObject[1] in object.lower():
+     foundObject = presetObjects[object]
+     break
+   objectLocation = argsObject[0]
+   loadedObject=objectLocation+foundObject
+   objects.remove(loadedObject)
+   desc.append(f"Object `{object}:{loadedObject}` was removed") 
```

Comment:
Updated walls so you can add them one at a time and added the new feature objects

Add and remove  for both